crates/transaction-emitter-lib/src/transaction_generator/publishing/module_simple.rs

```diff
@@ -261,16 +292,19 @@ fn minimize(module_id: ModuleId, other: AccountAddress) -> TransactionPayload {
     ])
 }
 
-fn make_or_change(rng: &mut StdRng, module_id: ModuleId) -> TransactionPayload {
+fn make_or_change(
+    rng: &mut StdRng,
+    module_id: ModuleId,
+    str_len: usize,
```

Comment:
Can you explain what is `str_len`  - is it the length of the key?

Reply:
resource has 3 fields:
        id: u64,
        name: String,
        data: Data,
        
 it is the lengths of the string :)